proto/BUILD.bazel

```
@@ -112,6 +116,12 @@ go_proto_compiler(
     ] + WELL_KNOWN_TYPE_RULES.values(),
 )
 
+go_non_go_exec_reset_target(
+    name = "protoc_reset_target_",
```

```suggestion
    name = "protoc",
```
maybe just name it "protoc"?